api/jobs.yaml

```diff
@@ -116,6 +131,45 @@ paths:
       tags:
         - Jobs
 definitions:
+  AuthenticationCapability:
+    description: Specification of the backend's authentication capabilities
+    required:
+      - required
```

Comment:
nit: I think I'd prefer to avoid required here and for the authentication tuple, i.e. if you don't care about authentication, you don't need to return anything.

Reply:
Good point, done.